doc/specs/OOBE-Dialog.md

```diff
@@ -0,0 +1,169 @@
+
+## 3. Requirements
+
+The base of this OOBE dialog was attributed by [Niel’s mock-up](https://github.com/microsoft/PowerToys/issues/1285).
+
+### 3.1. Functional Requirements
+
+#### 3.1.1. Functional Requirements Overview
+
+|No. | Requirement | Pri |
+| - | - | - |
+|1 | The OOBE Dialog window should launch immediately after installation. | P0 |
+|2 | The OOBE Dialog should consist of a collection of pages that describe the available PowerToys. | P0 |
+|3 | On launch, present the Welcome/Landing page in the OOBE Dialog window based on the template described in [Figure 5.1.1.](#511-oobe-dialog-layout) | P1 |
+|4 | Traverse the various OOBE Dialog pages via an list view options on the left-hand pane. | P0 |
+|5 | Alternatively traverse the various OOBE Dialog pages via an on-screen next/previous buttons on each page. | P1 |
+|6 | At any time, the user should be able to exit the OOBE dialog by closing the window. | P0 |
+|7 | Alternatively, the user should be able to exit the OOBE dialog by pressing the ESC key. | P1 |
+|8 | When the OOBE dialog is exited, save all the options that were set by the user. | P0 |
+|9 | In the PowerToys settings “General” menu, add a button to re-launch the OOBE dialog under the “Run at startup” option. | P0 |
+|10 | The OOBE dialog window should be able to be enlarged to accommodate accessibility needs. | P0 |
+|11 | The OOBE dialog should have a preset minimum window size to ensure all content can be viable with the need to scroll the page. | P1 |
+
+#### 3.1.2. Page Layout
+
+| No. | Requirement | Pri |
+| - | - | - |
+|1 | Each page of the OOBE dialog should focus on the details and settings of a single PowerToy. | P0 |
+|2 | A page should include textual information that gives an overview of its PowerToy's basic functions. | P0 |
+|3 | A page should include a hero image/GIF that provides a visual representation of its PowerToy's core functionality. | P0 |
+|4 | A page should include a link to that PowerToys overview page on Microsoft Docs. | P0 |
+|5 | For applicable PowerToys, include a &quot;Launch&quot; button on the page to allow users to instantiate the PowerToy directly from the OOBE dialog window. | P1 |
+|6 | Include a &quot;Settings&quot; button that deep links to the PowerToy's settings page. | P1 |
+|7 | A page should include a short-list of relevant settings options that the user can select between to customize their experience. | P1 |
+|8 | Include a &quot;Pro Tips&quot; section on the page that highlights a useful, yet often undiscovered feature for its PowerToy. | P3 |
+
+### 3.2. Displayed Options
+
+The proceeding subsections describe the settings we choose to display for each PowerToy and the justification for why it was included.
+
+We target only having at most 3 options per page/PowerToy to stay concise and ensure the settings displayed are the most useful for the general user-base.
+
+#### 3.2.1. Welcome/Landing Page
```

Comment:
Random idea: should we have a link to the release notes (or future "What's new" dialog here as well? E.g. for users that have used PowerToys in the past, removed it and want to see what's changed recently.

Reply:
Wouldn't hurt, we could probably get away with adding a link to release notes on the landing screen while we spec out how to incorporate the "What's new dialog" for SCOOBE updates.